package/src/completionCacheManager/completionCacheManager.ts

```diff
@@ -0,0 +1,22 @@
+import * as monaco from 'monaco-editor/esm/vs/editor/editor.api';
+import * as ls from 'vscode-languageserver-types';
+
+export const createCompletionCacheManager = (getFromLanguageService: GetFromLanguageService) => {
+    let completionList: Promise<ls.CompletionList> | undefined;
+    let lastWord: string | undefined;
+
+    return {
+        getCompletionItems: (word: string | undefined, resource: monaco.Uri, position: ls.Position) => {
+            const shouldGetItems = !lastWord || !word || !word?.includes(lastWord);
```

Comment:
We don't need the "?" after "word" in "!word.includes("

Reply:
You are completely right. I missed that. I will remove it in a separate PR.